assignments/hw3/calc.py

```diff
@@ -0,0 +1,54 @@
+"""Module calculates rolling batting averages
+    from a 'master' list of stats per game.
+"""
+
+import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import DataFrame, SparkSession
+
+ROLLING_SQL = """
```

Comment:
A comment explaining this query may be helpful:+1:

Reply:
Comment added in [`e172a9a`](https://github.com/timand32/bda696/pull/3/commits/e172a8a340ccb8ba32186cbd7a2b9fd5e2557ef7) (Thank you for catching this, I forgot to mention the code was adapted from the professor's query as well as giving a general explaination).